notification-rule-config-service-api/src/main/proto/org/hypertrace/notification/config/service/v1/notification_rule.proto

```diff
@@ -16,6 +16,15 @@ message NotificationRuleMutableData {
   string event_condition_id = 3;
   string event_condition_type = 4;
 
-  string channel_id = 5;
+  string channel_id = 5 [deprecated=true];
```

Comment:
we should have something like
```
oneof notification_target {
    NotificationChannel channel = 1;
    NotificationIntegrationTarget integration = 2;
  }

  message NotificationChannel {
    string channel_id = 1;
  }

  message NotificationIntegrationTarget {
    IntegrationType type = 1;
    string integration_id = 2;
  }

  enum IntegrationType {
    WIZ = 1;
  }
```

Reply:
@aaron-steinfeld Curious to know your thoughts on this.